### PR TITLE
Added GameResults

### PR DESCRIPTION
**Pull request by bras1223**

### CODE REVIEW

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            return;
+        }
+    }
+
+    /**
+     * Method that display the current settings of the room on the session view.
+     */
+    public void setGameInfo() {
+        Room room = currentSession.getRoom();
+        Game game = currentSession.getGame();
```
Comment (ghost) [inline Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java:94]: MAJOR Remove this unused "game" local variable. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1481)
MAJOR Remove this useless assignment to local variable "game". [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1854)

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+      
+        for(Player p : room.getTeamRed().getPlayers()) {
+            int goals = 0;
+            ResultTableItem r = new ResultTableItem(p.getUsername(), goals);
+            tblRedGoalsList.getItems().add(r);
+        }
+        //TODO: count goals per username.
+        
+        lblRoomName.setText("Game Results - " + room.getName());
+        //TODO: Set endscore
```
Comment (ghost) [inline Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java:110]: INFO Complete the task associated to this TODO comment. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1135)

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
+package nl.soccar.ui.fx.controller;
+
+import java.net.URL;
+import java.util.Optional;
+import java.util.ResourceBundle;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javafx.collections.FXCollections;
```
Comment (TehMessiah) [inline Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java]: MINOR Remove this unused import 'javafx.collections.FXCollections'. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+import java.util.Optional;
+import java.util.ResourceBundle;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javafx.collections.FXCollections;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListView;
```
Comment (TehMessiah) [inline Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java]: MINOR Remove this unused import 'javafx.scene.control.ListView'. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+            ResultTableItem r = new ResultTableItem(p.getUsername(), goals);
+            tblBlueGoalsList.getItems().add(r);
+        }
+      
+        for(Player p : room.getTeamRed().getPlayers()) {
+            int goals = 0;
+            ResultTableItem r = new ResultTableItem(p.getUsername(), goals);
+            tblRedGoalsList.getItems().add(r);
+        }
+        //TODO: count goals per username.
```
Comment (ghost) [inline Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java:107]: INFO Complete the task associated to this TODO comment. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1135)

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+     * Method that navigates to the game view and set the application window to
+     * full screen mode.
+     */
+    public void startGame() {
+        Main main = Main.getInstance();
+        main.setScene(FXMLConstants.LOCATION_GAME_VIEW);
+        main.setFullScreen(true);
+    }
+    
+    public void closeResults() {
```
Comment (TehMessiah) [inline Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 8 issues
* MAJOR 2 major
* MINOR 4 minor
* INFO 2 info

Watch the comments in this conversation to review them.

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java (hunk 1 above)
Comment (bras1223) [reply Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java:94]: Game will be used during implementation of TODO's.

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java (hunk 2 above)
Comment (bras1223) [reply Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java:110]: TODO's will be implemented after push of Marc's ScoreBoard utils.

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java (hunk 5 above)
Comment (bras1223) [reply Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java:107]: TODO's will be implemented after push of Marc's ScoreBoard utils.

**Comment by ghost** [comment]
SonarQube analysis reported 4 issues
* MAJOR 2 major
* INFO 2 info

Watch the comments in this conversation to review them.

**Review by BeeHiveJava — APPROVED** [review]

**Pull request merged**